ldk/node/package-lock.json

```diff
@@ -1,8 +1,10679 @@
 {
   "name": "@oliveai/ldk",
-  "version": "2.0.0-alpha",
-  "lockfileVersion": 1,
+  "version": "2.0.0-beta",
```

Comment:
What happened here?

Reply:
Not sure actually, looks like it decided to move a line down.